### PR TITLE
Check for Neutralizing Gas in tooltips

### PR DESCRIPTION
**Pull request by Karthik99999**

### CODE REVIEW

**Comment by DaWoblefet** [comment]
What does this do?

**Comment by Karthik99999** [comment]
> What does this do? Is this the Skill Swap not displaying the correct Ability thing?

Mainly accounts for Neutralizing Gas. I noticed that No Guard would still show up on the move tooltip when Neutralizing Gas was up.

File: src/battle-tooltips.ts [hunk 1]
```diff
@@ -62,6 +62,7 @@ class ModifiableValue {
 			this.comment.push(` (${abilityName} suppressed by Gastro Acid)`);
 			return false;
 		}
+		if (!this.pokemon?.effectiveAbility(this.serverPokemon)) return false;
```
Comment (Zarel) [inline src/battle-tooltips.ts:65]:
```suggestion
		// Check for Neutralizing Gas
		if (!this.pokemon?.effectiveAbility(this.serverPokemon)) return false;
```

**Comment by Zarel** [comment]
https://github.com/smogon/pokemon-showdown/blob/master/CONTRIBUTING.md

> CM1. What, not how
>
> Commits should describe what the code _does_, not how it does it.

;)

**Comment by Karthik99999** [comment]
> https://github.com/smogon/pokemon-showdown/blob/master/CONTRIBUTING.md
> 
> 
> 
> > CM1. What, not how
> 
> >
> 
> > Commits should describe what the code _does_, not how it does it.
> 
> 
> 
> ;)

You would think after all my PRs I would understand this lol

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**